doc/DESI_SPECTRO_DATA/index.rst

```diff
@@ -2,6 +2,8 @@
 DESI_SPECTRO_DATA
 =================
 
+Default $DESI_ROOT/spectro/data
```

Comment:
Is the pipe symbol (leading `|`) in the other files meaningful? If so, why is it not applied to the start of this line?

Reply:
The pipe `|` is used in rst when you want several lines with exactly the line breaks as you typed them but without creating a new paragraph, thus it's not needed for a single line.

Reply:
You're right. I think it means something very different in MD.